src/screen/ExportAccountWithJsonFileScreen.tsx

```diff
@@ -0,0 +1,113 @@
+import IdentityIcon from '@polkadot/reactnative-identicon/Identicon';
+import keyring from '@polkadot/ui-keyring';
+import {RouteProp} from '@react-navigation/core';
+import {Icon, Input, Text, ListItem, useTheme, Button} from '@ui-kitten/components';
+import FormLabel from 'presentational/FormLabel';
+import Padder from 'presentational/Padder';
+import SafeView, {noTopEdges} from 'presentational/SafeView';
+import React from 'react';
+import {View, StyleSheet, TouchableOpacity} from 'react-native';
+import {AccountsStackParamList} from 'src/navigation/navigation';
+import {exportAccountWithJsonFileScreen} from 'src/navigation/routeKeys';
+import {monofontFamily, standardPadding} from 'src/styles';
+import Share from 'react-native-share';
+
+export function ExportAccountWithJsonFileScreen({
+  route,
+}: {
+  route: RouteProp<AccountsStackParamList, typeof exportAccountWithJsonFileScreen>;
+}) {
+  const theme = useTheme();
+  const {address} = route.params;
+  const pair = keyring.getPair(address);
```

Comment:
This throws an error if it does not find the pair. We will probably prevent navigating to this screen if it's an external account but it does not hurt to wrap this in a try/catch just in case